Formula/kotlin-cli-starter.rb

```diff
@@ -0,0 +1,27 @@
+class KotlinCliStarter < Formula
+  desc "Git extension to generate reports for standup - in Kotlin Multiplatform"
+  homepage "https://github.com/jmfayard/kotlin-cli-starter"
+  url "https://github.com/jmfayard/kotlin-cli-starter/archive/refs/tags/v0.3.tar.gz"
+  sha256 "bbfaec0b9ed0918feeeed02036e8bddfd5d028f11e09fec8e9af1361a3eaeace"
+  license "MIT"
+
+  depends_on "gradle" => :build
+  depends_on "openjdk" => :build
+
+  conflicts_with "git-standup", because: "it's a reimplementation of this tool"
```

Comment:
```suggestion
  conflicts_with "git-standup", because: "both install `git-standup` binaries"
```
For consistency with other formulae.